src/Http/Authentication.Abstractions/src/AuthenticationOptions.cs

```diff
@@ -95,5 +95,10 @@ public void AddScheme(string name, Action<AuthenticationSchemeBuilder> configure
         /// If true, SignIn should throw if attempted with a ClaimsPrincipal.Identity.IsAuthenticated = false.
         /// </summary>
         public bool RequireAuthenticatedSignIn { get; set; } = true;
+
+        /// <summary>
+        /// If true, claims transformation should only be applied once per ClaimsPrincipal per request.
+        /// </summary>
+        public bool ApplyClaimsTransformationOnce { get; set; } = true;
```

Comment:
This is a weird flag. Is this a bug fix or not?

Reply:
This started as a bug fix, we could just remove it now I guess?  Then they would have to replace the AuthenticationService itself to turn this off, but seems fairly unlikely that anyone wants the old behavior

Reply:
Go ahead and remove the flag.